api/commands/run.ts

```diff
@@ -16,16 +16,7 @@ const isSuccessfulAction = (actionResult: dataform.IActionResult) =>
 
 export function run(graph: dataform.IExecutionGraph, credentials: Credentials): Runner {
   const dbadapter = dbadapters.create(credentials, graph.projectConfig.warehouse);
-  const runner = Runner.create(dbadapter, graph);
-  const executeAndCloseDbAdapter = async () => {
```

Comment:
Why remove this bit? Seems important.

Reply:
I have just moved this code into execute()/result(), so that we don't have unhandled promises.